docs/network_disruption/cloud-managed-services.md

```diff
@@ -39,9 +39,16 @@ spec:
         - service: "S3"
           flow: "egress" # available are egress or ingress. Optional
           protocol: "tcp" # available are tcp or udp. Optional
+          connState: "new"
```

Comment:
We should be explicit that `connState` is optional, otherwise it looks required given every other optional field is marked as such